web/src/app/services/UniversalKey/UniversalKeyDefaultActions.tsx

```diff
@@ -0,0 +1,78 @@
+import { Button, Card } from '@mui/material'
+import React, { Suspense, useState } from 'react'
+import FlatList from '../../lists/FlatList'
+import { Edit } from '@mui/icons-material'
+import DefaultActionEditDialog from './DefaultActionEditDialog'
+import UniversalKeyActionsList from './UniversalKeyActionsList'
+import { gql, useQuery } from 'urql'
+import { IntegrationKey } from '../../../schema'
+
+interface UniversalKeyDefaultActionProps {
+  serviceID: string
+  keyID: string
+}
+
+const query = gql`
+  query UniversalKeyPage($keyID: ID!) {
+    integrationKey(id: $keyID) {
+      id
+      config {
+        defaultActions {
+          dest {
+            type
+            args
+          }
+          params
+        }
+      }
+    }
+  }
+`
+
+export default function UniversalKeyDefaultActions(
+  props: UniversalKeyDefaultActionProps,
+): React.ReactNode {
+  const [edit, setEdit] = useState(false)
+  const [q] = useQuery<{ integrationKey: IntegrationKey }>({
```

Comment:
nit, for clarity:

name `q` something such as status, or destructure them inplace, e.g. const [{ data, fetching, error }, commit]